engines/account/app/controllers/account/api/v1/login_controller.rb

```diff
@@ -16,7 +17,7 @@ def login
         private
 
         def login_params
-          params.permit(:email, :password)
+          params.permit(:username, :email, :password)
```

Comment:
yes and then how will you compose the query looking for a user by username OR email?